website/docs/docs/dbt-versions/release-notes/08-May-2023/discovery-api-public-preview.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Latest environment state in the dbt Cloud Discovery API"
+id: "discovery-api-public-preview"
+description: "Public preview of the dbt Cloud Discovery API is now available."
+sidebar_label: "New: Latest environment state in the Discovery API"
+tags: [May-24-2023, API]
```

Comment:
Can you add a `sidebar_position: 5` to the header so it shows up above The [other one](https://github.com/dbt-labs/docs.getdbt.com/blob/current/website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md). THis is a docusaurus thing and I think it's written down. I can verify if it's in our docs process later!

Reply:
totally! didn't know about `sidebar_position` . thank you!